lib/model/cardinalPortPosition.js

```diff
@@ -15,6 +17,7 @@ var CardinalPortPosition = function(direction,percentage) {
  * @function getDirection
  */
 CardinalPortPosition.prototype.getDirection = function() {
+  "use strict";
```

Comment:
I believe we can put "use strict"; at file-level scope and it should work that was as well. I can change it so that it uses that instead if it helps.